packages/frontend-app-api/src/tree/resolveAppNodeSpecs.ts

```diff
@@ -209,18 +209,19 @@ export function resolveAppNodeSpecs(options: {
         Boolean(existing.params.disabled) !== Boolean(overrideParam.disabled)
       ) {
         existing.params.disabled = Boolean(overrideParam.disabled);
-        if (!existing.params.disabled) {
-          // bump
-          configuredExtensions.splice(existingIndex, 1);
-          configuredExtensions.push(existing);
-        }
       }
+      order.set(extensionId, existing);
```

Comment:
Hmm, would it be enough to move out the existing `// bump` logic to here? avoiding the extra Map. Shouldn't really matter if it's disabled or not either afaik

Reply:
Tricky thing is when you have a thing be mentioned more than once in config. I order them by first occurrence, not last

Reply:
I did do something like that from the start btw. With an unshift, and iterating the settings in reverse order etc. Didn't work as well and was no less complex.

Reply:
Aight, :shipit: